browser/android/speedreader/speedreader_utils.cc

```diff
@@ -0,0 +1,30 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/android/speedreader/speedreader_utils.h"
+
+#include "base/android/jni_android.h"
+#include "base/logging.h"
+#include "brave/browser/speedreader/speedreader_tab_helper.h"
+#include "content/public/browser/web_contents.h"
+#include "jni_headers/brave/build/android/jni_headers/BraveSpeedReaderUtils_jni.h"
```

Comment:
CI complains as this path isn't correct, it should be
`#include "brave/build/android/jni_headers/BraveSpeedReaderUtils_jni.h"`

Reply:
This is strange because on my local build that is the right path. Not sure why.